packages/api-svc/test/unit/settlement_api_svc_route.test.ts

```diff
@@ -0,0 +1,985 @@
+                debitBalance: "10",
+                creditBalance: "0"
+            },
+            {
+                id: "SMBA002",
+                participantId: "FSP-B",
+                accountExtId: "string",
+                debitBalance: "0",
+                creditBalance: "10"
+            }
+        ];
+
+        //Prepare mocked Settlement Matrix Participant Balances
+        mockedSettlementMatrixParticipantBalances = [
+            {
+                participantId: "FSP-A",
+                debitBalance: "10",
+                creditBalance: "0"
+            },
+            {
+                participantId: "FSP-B",
+                debitBalance: "0",
+                creditBalance: "10"
+            }
+        ]
+
+        //Prepare mocked Settlement Matrix batch
+        mockedSettlementMatrixBatches = [{
+            id: "DEFAULT.USD.2023.06.19.08.30.001",
+            name: "DEFAULT.USD.2023.06.19.08.30",
+            batchDebitBalance: "10",
+            batchCreditBalance: "10",
+            state: "OPEN",
+            batchAccounts: mockedSettlementMatrixBatchAccount
+        }]
+
+        //Prepare mocked Matrix
+        mockedSettlementMatrix = {
+            id: "SM001",
+            createdAt: new Date().getTime() / 1000,
+            updatedAt: 0,
+            dateFrom: null,
+            dateTo: null,
+            currencyCode: "USD",
+            settlementModel: "DEFAULT",
+            batches: mockedSettlementMatrixBatches,
+            participantBalances: mockedSettlementMatrixParticipantBalances,
+            participantBalancesDisputed: [],
+            state: "SETTLED",
+            type: "STATIC",
+            generationDurationSecs: 2,
+            totalDebitBalance: "10",
+            totalCreditBalance: "10",
+            totalDebitBalanceDisputed: "0",
+            totalCreditBalanceDisputed: "0"
+        }
+
+
+    })
+
+    afterAll(async () => {
+        await Service.stop();
+    });
+
+    /**Happy-path Test Cases */
+
+    test("GET /batches/:id - should fetch settlement batch by id", async () => {
+
+        //Arrange
+        let mockBatches: ISettlementBatch[] = [
+            {
+                id: "DEFAULT.USD.2023.06.19.08.30.001",
+                timestamp: Date.now(),
+                settlementModel: 'DEFAULT',
+                currencyCode: 'USD',
+                batchName: "DEFAULT.USD.2023.06.19.08.30",
+                batchSequence: 2, //100 EURO
+                state: "OPEN",
+                accounts: []
+            },
+            {
+                id: "DEFAULT.USD.2023.06.20.01.20.001",
+                timestamp: Date.now(),
+                settlementModel: 'CBX',
+                currencyCode: 'USD',
+                batchName: "DEFAULT.USD.2023.06.20.01.20",
+                batchSequence: 1, //100 EURO
+                state: "CLOSED",
+                accounts: []
+            }
+        ];
+
+
+        await mockBatchRepo.storeNewBatch(mockBatches[0]);
```

Comment:
Same, lets add `storeNewBatch` to support storing multiple batches via spread operator?